docs/why-logfire/python-centric.md

```diff
@@ -0,0 +1,17 @@
+# Python-centric insights :material-snake:
+
+Pydantic Logfire automatically instruments your code for minimal manual effort, provides exceptional insights into async code, offers detailed performance analytics, and displays Python objects the same as the interpreter. Pydantic Logfire  gives you a clearer view into how your Python is running than any other observability tool.
```

Comment:
```suggestion
Pydantic Logfire automatically instruments your code for minimal manual effort, provides exceptional insights into async code, offers detailed performance analytics, and displays Python objects the same as the interpreter. Pydantic Logfire gives you a clearer view into how your Python is running than any other observability tool.
```

Do we mean to consistently use **Pydantic Logfire**? Wasn't sure, had seen in some other places.